test_pull_requests

```diff
@@ -99,14 +99,18 @@ config = args[:config]
 merge_pretest_success = args[:merge_pretest_success]
 dry_run = args[:dry_run]
 
-PROPERTIES_LOCATION = File.expand_path(config || DEFAULT_PROPERTIES_LOCATION)
-Properties = if File.exists? PROPERTIES_LOCATION
-  JSON.parse(IO.read(PROPERTIES_LOCATION))
-else
-  $stderr.puts "Properties file not found: #{PROPERTIES_LOCATION}"
-  exit 255
+def load_properties
+  if File.exists? PROPERTIES_LOCATION
```

Comment:
This is another place where it seems of dubious value to be coding this -- what happens when you `IO.read()` a file that doesn't exist? Is it not an error message? Doesn't it exit?